en/docs/setup/deployment-patterns.md

```diff
@@ -204,42 +113,33 @@ data_source = "jdbc/WSO2UserDS"
 
 WSO2 Identity Server comprises of three different registry repositories.
 
-1. **Local Repository**: Used to store configuration and runtime data that is local to the server.
+1. **Local Repository**: Use to store configuration and runtime data that is local to the server.
 
-2. **Configuration Repository**: Used to store product-specific configurations. 
+2. **Configuration Repository**: Use to store product-specific configurations. 
 
-3. **Governance Repository**: Used to store configuration and data that are shared across the whole platform. This typically includes services, service descriptions, endpoints or data sources.
+3. **Governance Repository**: Use to store configuration and data that are shared across the whole platform. This typically includes services, service descriptions, endpoints or data sources.
```

Comment:
This is used to xxxx.

Reply:
Resolved: e544b4e2c4470945228282043318f840705e1712